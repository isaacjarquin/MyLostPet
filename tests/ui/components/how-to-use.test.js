import "react-native"
import React from "react"
import HowToUse from "../../../components/how-to-use"

import renderer from "react-test-renderer"

it("renders correctly", () => {
    const tree = renderer.create(
        <HowToUse />
    ).toJSON()
    expect(tree).toMatchSnapshot()
})
