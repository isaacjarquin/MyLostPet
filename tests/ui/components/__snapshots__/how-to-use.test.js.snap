// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "grey",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "height": "100%",
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <RCTScrollView
      style={
        Object {
          "backgroundColor": "black",
          "opacity": 0.8,
        }
      }
    >
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "alignSelf": "center",
              "backgroundColor": "transparent",
              "color": "white",
              "fontSize": 20,
              "fontWeight": "bold",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          Como usar la aplicación
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          El objetivo principal de la aplicación es poner en contacto a la persona que haya perdido su mascota con la persona que la haya encontrado.
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          La idea principal radica en que cuando nos encontramos un animal abandonado, todo lo que tenemos que hacer es introducir los datos relativos al animal y algunos datos personales de contacto. Tus datos personales en ningún momento se mostrarán a terceros ni serán utilizados para enviarte publicidad. La única finalidad de los mismos es la de poder comunicarte con la persona que haya perdido a su mascota. Una vez que hayas introducido los datos de la mascota encontrada, junto con una foto, que deberá ser inferior a 1Mbyte, debes hacer click en el botón guardar datos, y los datos serán almacenados en nuestra base de datos.
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          Si por el contrario, lo que deseas es encontrar a tu mascota perdida, debes usar el filtro que se desplegara al pulsar el botón 'Buscar mascota' en la parte inferior de la página. Aquí podrás filtrar por tipo de mascota, comunidad autónoma y provincia en la que se a extraviado tu mascota. Esto te mostrará una lista con todas las mascotas perdidas en la provincia seleccionada que coinciden con el filtro aplicado. Debido a que esta lista podría ser muy grande en determinados casos, hemos añadido un segundo nivel de filtro que te permitiran hacer una busqueda aun mas selectiva, filtrando por ciudad o municipio, raza y tamaño. La combinación de estos 6 filtros nos permitirá una potente búsqueda que debería ser suficiente para encontrar a tu mascota. Una vez hayas encontrado a tu mascota, debes hacer click sobre ella lo cual te direccionara a una pantalla donde podrás ver información relativa al animal en question.
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "margin": 10,
              "marginBottom": 10,
            }
          }
        >
          Por último si haces click en el botón 'Contactar' se desplegará un formulario en el que deberás introducir tu información personal de contacto, para que la persona que encontró tu mascota se pueda poner en contacto contigo.
        </Text>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
